Parser/tokenizer.c

```
@@ -2496,41 +2497,42 @@ tok_get_normal_mode(struct tok_state *tok, tokenizer_mode* current_tok, struct t
     case ')':
     case ']':
     case '}':
-        if (!tok->level) {
+        if (!tok->ignore_unmatched_parens && !tok->level) {
```

Here we are also ignoring errors inside f-string. Is that intended?